### DISCUSSION TITLE
Allure reports - way to surface the WDIO commands instead of displaying HTTP requests

### DISCUSSION THREAD

**Discussion by steviewangder** [webdriverio/webdriverio#12926] [Q&A]
In the console, I can see WDIO commands being executed like the following:

![image](https://github.com/webdriverio/webdriverio/assets/82411557/3559b9f7-f2e7-4369-b1da-46f50be97b96)

On Allure reports, these are displayed as HTTP commands to the server. Is there a way to display the Webdriver commands on the report instead?

![image](https://github.com/webdriverio/webdriverio/assets/82411557/376abcc4-828e-486e-b63a-b200c0bd52d2)

**Comment by christian-bromann** [comment]
@steviewangder thanks for raising the issue, I created a feature request in #12927 .. any contributions are appreciated!

**Comment by sudharsan-selvaraj** [comment]
@christian-bromann Hey, I'd be glad to handle this. Let's discuss the expectations. Should we replace the W3C endpoint with the command name, or should we simply append the command name to the end of the URL?

**Reply by christian-bromann** [reply]
Maybe we can include the command name, if possible and the W3C endpoint in `(` and `)`.

**Comment by sudharsan-selvaraj** [comment]
@christian-bromann @steviewangder Any thoughts? Open for suggestions.

<img width="754" alt="Screenshot 2024-06-05 at 8 50 39 PM" src="https://github.com/webdriverio/webdriverio/assets/20136913/5e96df74-e804-48cc-881b-439453e0c2f6">

**Reply by christian-bromann** [reply]
That looks great to me 💯

**Reply by sudharsan-selvaraj** [reply]
Awesome, i have created a [PR](https://github.com/webdriverio/webdriverio/pull/12982) with the changes and kindly review it.
